circleci: failed tests (14666):

The following test appears to have failed:

[#14666](https://circleci.com/gh/cockroachdb/cockroach/14666):

```
go test -v  -i ./...
go test  -run . -timeout 1m10s ./... -v --verbosity=1 --vmodule=monitor=2,tracer=2
?   	github.com/cockroachdb/cockroach	[no test files]
panic: sql: Register called twice for driver postgres

goroutine 1 [running]:
panic(0x11d3c40, 0xc820201940)
	/usr/local/go/src/runtime/panic.go:464 +0x3e6
database/sql.Register(0x15118f0, 0x8, 0x7f4905b8bd88, 0x21c97d8)
	/usr/local/go/src/database/sql/sql.go:45 +0x181
github.com/lib/pq.init.1()
	/go/src/github.com/lib/pq/conn.go:44 +0x71
github.com/lib/pq.init()
	/go/src/github.com/lib/pq/user_posix.go:24 +0x4b0
github.com/cockroachdb/cockroach/acceptance.init()
	/go/src/github.com/cockroachdb/cockroach/acceptance/util_test.go:297 +0xe2
main.init()
	github.com/cockroachdb/cockroach/acceptance/_test/_testmain.go:82 +0x45
FAIL	github.com/cockroachdb/cockroach/acceptance	0.038s
?   	github.com/cockroachdb/cockroach/acceptance/cluster	[no test files]
?   	github.com/cockroachdb/cockroach/acceptance/terrafarm	[no test files]
=== RUN   TestClientSSLSettings
I160316 21:04:44.072115 context.go:127  no certificates directory specified: using insecure TLS
--- PASS: TestClientSSLSettings (0.01s)
=== RUN   TestServerSSLSettings
--- PASS: TestServerSSLSettings (0.02s)
PASS
ok  	github.com/cockroachdb/cockroach/base	0.044s
=== RUN   TestQuit

```
Please assign, take a look and update the issue accordingly.

--- a/acceptance/util_test.go
+++ b/acceptance/util_test.go
@@ -33,13 +33,13 @@ import (
 
 	"github.com/docker/engine-api/types"
 	"github.com/docker/engine-api/types/container"
-	_ "github.com/lib/pq"
 
 	"github.com/cockroachdb/cockroach/acceptance/cluster"
 	"github.com/cockroachdb/cockroach/acceptance/terrafarm"
 	"github.com/cockroachdb/cockroach/base"
 	"github.com/cockroachdb/cockroach/util/caller"
 	"github.com/cockroachdb/cockroach/util/log"
+	_ "github.com/cockroachdb/pq"
 )
 
 var flagDuration = flag.Duration("d", cluster.DefaultDuration, "duration to run the test")

--- a/sql/pgbench/cmd/pgbenchsetup/main.go
+++ b/sql/pgbench/cmd/pgbenchsetup/main.go
@@ -23,9 +23,8 @@ import (
 	"net/url"
 	"os"
 
-	_ "github.com/lib/pq"
-
 	"github.com/cockroachdb/cockroach/sql/pgbench"
+	_ "github.com/cockroachdb/pq"
 )
 
 var usage = func() {

